packages/cli/src/lib/__tests__/mergeBasics.test.js

```diff
@@ -0,0 +1,141 @@
+import { merge } from '../merge'
+import { concatUnique } from '../merge/strategy'
+
+import { unindented } from './fixtures/unindented'
+
+const expectMerged = (base, ext, merged, strategy = {}) => {
+  expect(merge(unindented(base), unindented(ext), strategy)).toBe(
+    unindented(merged)
+  )
+}
+
+const expectTrivialConcat = (base, ext, strategy = {}) => {
+  const ubase = unindented(base)
+  const uext = unindented(ext)
+  expect(merge(ubase, uext, strategy)).toBe(`${ubase}\n${uext}\n`)
+}
+
+describe('the basics', () => {
+  it('Inserts extension declarations in the last possible position', () => {
+    // Notice how 'y' is reordered above 'list' to ensure its reference in 'list' is valid.
+    expectMerged(
+      "const x = 'x'\nconst list = [x]",
+      "const y = 'y'\nconst list = [y]",
+      `\
+      const x = 'x'
+      const y = 'y'
+      const list = [x, y]
+      `,
+      { ArrayExpression: concatUnique }
+    )
+  })
+})
+
+describe('Import behavior', () => {
+  it('keeps both identical namespace imports', () => {
```

Comment:
I don't understand this test case. When would you ever want to keep both these import statements?